documentation/leo/03_language.md

```diff
@@ -267,8 +271,10 @@ transition foo(public a: field) -> field {
 ### Helper Function
 
 A helper function is declared as `function {name}() {}`.
-Helper functions contain expressions and statements that can compute values.
-Helper functions that cannot be executed directly. Helper functions must be called by other functions.
+Helper functions contain expressions and statements that can compute values,
+however helper functions cannot produce `records`.
+
+Helper functions that cannot be executed directly. Instead, they must be called by other functions.
```

Comment:
There's a stray 'that'.

You may also consider rephrasing the first sentence as "Helper functions cannot be called directly from the outside." (Not sure which one is better though.)